generator/templates/default/src/plugins/vuetify.js

```diff
@@ -12,7 +12,7 @@ import {
   VToolbar,
   transitions
 } from 'vuetify'
-import '../node_modules/vuetify/src/stylus/app.styl'
+import '../..//node_modules/vuetify/src/stylus/app.styl'

```

Comment:
remove extra `/`

Reply:
Shouldn't this just be `'vuetify/src/stylus/app.styl'`? No idea why you guys like relative paths so much